docs/UserGuide.md

```diff
@@ -13,11 +13,173 @@
+
+Instruction: 
+1. `cat CATEGORY_NAME`   
+  
+e.g. `cat entertainment`, `cat stocks`
+
+Expected output:   
+![Bookmark_1.3](Images/BookmarkUG/Figure3.png)
+
+**Useful Information:** 
+* You can only use this command anywhere after entering the bookmark mode.  (Refer to General Command 2)
+* NUS, Zoom, Internship, Hackathon and Career Talk categories are included in the default.
+
+If you encounter any errors, take note of the following warnings. If problems still persist, refer to FAQ at the end of the user guide. 
+
+**Warning:** 
+* Ensure that the category name is not empty. Error will be shown if the category name is empty. 
+
+###1.4 Deleting a category :  
+`delete CATEGORY_NUMBER`  
+
+When you have a category, but you have no important links related to the category, take it out of your bookmark list! Organise and design your bookmark categories to your own personal needs by adding your own category (refer to 1.3) and deleting unnecessary categories using these following commands.
+
+Instruction:  
+
+1. delete CATEGORY_NUMBER  
+
+e.g. delete 6  
+Expected output:  
+![Bookmark_1.4](Images/BookmarkUG/Figure4.png)
+
+**Useful Information:**
+* You can only use this command in bookmark main.  (Refer to 1.5 to go back to the bookmark main)
+
+Before deleting a category, take note of the following warnings. If you encounter any problems, refer to FAQ at the end of the user guide. 
+
+**Warning:**
+* Ensure that the category NUMBER that you have chosen is valid. Error will be shown if the category NUMBER does not exist or the NUMBER you have entered is not a number. To find out the valid category NUMBER, go back to bookmark main. (Refer to 1.5)
+* CAUTION: Deleting a category will delete all the bookmarks in the category! 
+
+###1.5 Going back to bookmark main : 
+`back` 
+There are two modes in bookmark feature: the main bookmark mode (refer to General Command 2) and the category mode (refer to 1.1). To navigate freely from mode to mode, use the following command. This command goes back to the previous mode depending on which mode you are in. 
+ 
+Instruction: 
+1. back 
+
+Expected output:   
+![Bookmark_1.5](Images/BookmarkUG/Figure5.png)
+![Bookmark_1.6](Images/BookmarkUG/Figure6.png)
+
+**Useful Information:**
+* When you input back in any bookmark category, you are brought back to the bookmark main. 
+* When you input back in the bookmark main, you will be prompted to type “exit” to exit bookmark mode. 
+
+###1.6 Adding a link to your chosen category : 
+`add LINK` 
+If you have a link to add to your bookmark category, follow these commands. Make sure that you have chosen your category (Refer to 1.1)  and ensure that you do not add an invalid link or an empty link to your bookmark.
```

Comment:
Remember to add an extra line after `add LINK` for it to show as the next paragraph